lib/mongoid/contextual/memory.rb

```diff
@@ -110,9 +110,24 @@ def each
       # @example Do any documents exist for the context.
       #   context.exists?
       #
+      # @example Do any documents exist for given _id.
+      #   context.exists?(BSON::ObjectId(...))
+      #
+      # @example Do any documents exist for given conditions.
+      #   context.exists?(name: "...")
+      #
+      # @param [ Hash | Object | false ] id_or_conditions an _id to
+      #   search for, a hash of conditions, or false.
+      #
       # @return [ true | false ] If the count is more than zero.
-      def exists?
-        any?
+      #   Always false if passed false.
+      def exists?(id_or_conditions = nil)
+        case id_or_conditions
+        when nil then any?
```

Comment:
Per https://github.com/rails/rails/commit/d48dd18bb2a3d0c46708a9ee217909783b997cb2 I think this should return false (and this case added to docs).

Reply:
I don't know if I agree... Then we would have to change the default parameter?